java-numbers-2/pom.xml

```diff
@@ -21,6 +21,11 @@
             <artifactId>jmh-generator-annprocess</artifactId>
             <version>${jmh-generator.version}</version>
         </dependency>
+        <dependency>
```

Comment:
is this a new dependency added here?

Reply:
This dependency was present. I have moved the version into properties tag.